js/scripts.js

```diff
@@ -0,0 +1,14 @@
+var allSpans = $("span");
+    allSpans.each(function(index, element) {
+    	$("span:even").css('color', 'red');
```

Comment:
bardzo nieefektywne - zobacz, że jak będziesz miał 50 spanów w `allSpan`, to 50 razy w pętli mówisz mu "znajdź wszystkie parzyste spany i daj im kolor red" - jakby to było raz poza pętlą zrobione to OK, ale robisz to 50 x w pętli. Jeśli w pętli chcesz to robić to sprawdzaj czy `index` dzielony na 2 daje reszte, jeśli tak to jest nieparzysty. Jeśli jest parzysty to `element` powinien dostać kolor red.